server/src/main/java/org/apache/druid/server/coordinator/rules/PeriodBroadcastDistributionRule.java

```diff
@@ -94,7 +104,9 @@ public boolean equals(Object o)
     if (!Objects.equals(period, that.period)) {
       return false;
     }
-
+    if (!Objects.equals(includeFuture, that.includeFuture)) {
```

Comment:
`includeFuture` can't be null. `if (includeFuture != that.includeFuture)`?

Reply:
Good, I'll change it.